tests/helpers.ts

```diff
@@ -18,7 +19,44 @@ export function sampleAddons() {
     .filter((el) => fs.existsSync(join(samplesDir, el, 'input.json')));
 }
 
-export async function addonFrom(fixture: string): Promise<Project> {
+function addDependency(nodeModulesPath: string, name: string, version: string) {
```

Comment:
can you add a comment to this about getting around an issue with fixturify? this doesn't _actually_ add the dependency (and its contents), it just fakes it

Reply:
updated